### ISSUE TITLE
Cannot generate icon cache

### ISSUE THREAD

**Issue by SuNNjek** [B00merang-Artwork/Windows-XP#8]
When I install the theme, XFCE complains that the icon theme has no cache and tells me to regenerate it using
```
gtk-update-icon-cache /usr/share/icons/Windows-XP-3.1/
```

However, this fails with the message:
```
gtk-update-icon-cache: The generated cache was invalid.
```

**Comment by Elbullazul** [comment]
yeah, it's a recurrent bug, but there's little info on how to fix it

**Comment by dylansrogers** [comment]
There are 25 files in the icons directory that have spaces in their names. You must replace the space with a "_" or something similar, then generate the cache.
